package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "dcc-docs",
+  "version": "1.0.0",
+  "description": "Documentation for ICGC DCC using [mkdocs](http://www.mkdocs.org/) and GitHub Pages hosted at [http://icgc-dcc.github.io/dcc-docs](http://icgc-dcc.github.io/dcc-docs)",
+  "main": "none",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "build": "npm run build:scripts && npm run build:styles && npm run build:mkdocs",
+    "build:prod": "npm run build:scripts:prod && npm run build:styles:prod",
+    "build:scripts": "webpack --config config/webpack.js --progress --profile --colors --display-error-details --display-cached",
+    "build:scripts:prod": "NODE_ENV=production npm run build:scripts",
+    "build:mkdocs": "mkdocs build",
+    "build:styles": "mkdir -p theme/css-out && stylus src/styles --include node_modules --include-css --out theme/css-out",
+    "build:styles:prod": "npm run build:styles -- --compress",
+    "start": "npm run dev",
+    "dev": "npm run dev:scripts & npm run dev:stylus & npm run dev:mkdocs",
+    "dev:scripts": "webpack --config config/webpack.js --watch --progress --profile --colors --display-error-details --display-cached",
+    "dev:mkdocs": "mkdocs serve",
+    "dev:stylus": "npm run build:styles -- --watch",
+    "publish": "npm run build:prod && mkdocs gh-deploy --clean"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/icgc-dcc/dcc-docs.git"
+  },
+  "author": "",
+  "license": "UNLICENSED",
+  "bugs": {
+    "url": "https://github.com/icgc-dcc/dcc-docs/issues"
+  },
+  "homepage": "https://github.com/icgc-dcc/dcc-docs#readme",
+  "devDependencies": {
+    "@icgc/dictionary-viewer": "1.0.3",
```

Comment:
@xxxxx/yyyyyy denotes a scoped package named yyyyy belonging to the user/organization xxxxx

Scoped packages allow us to not pollute the global npm namespace with whatever names we choose, and also allow us to name our packages whatever without having to worry about whether the name has already been taken :)